app/main.py

```diff
@@ -1,8 +1,62 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
```

Comment:
these values have specific range of correct values, let's check if it's correct. If not -> set some default correct value

Reply:
Please set the default value instead of raising exception